specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimtenant.json

```diff
@@ -34,6 +34,59 @@
     }
   },
   "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/tenant": {
```

Comment:
>tenant [](start = 128, length = 6)

the types are suppose to follow plural version. Looks like this was not caught in the initial versions.

Reply:
We didn't have "/tenant" collection at all, we had only /tenant/access endpoint.
Same question: Please advise what is the proper way to implement GET endpoint for the entity without collection.